agents-api/agents_api/workflows/task_execution/__init__.py

```diff
@@ -437,7 +437,7 @@ async def run(
             ]["type"] == "integration":
                 workflow.logger.debug("Prompt step: Received INTEGRATION tool call")
 
-                # FIXME: Implement integration tool calls
+                # TODO: Implement integration tool calls
```

Comment:
@whiterabbit1983 we need to implement these 3 blocks as well, refer to the `ToolStep(...)` for how to do them